forge-gui/res/cardsfolder/upcoming/dark_apostle.txt

```diff
@@ -0,0 +1,10 @@
+Name:Dark Apostle
+ManaCost:3 R
+Types:Creature Astartes Warlock
+PT:3/3
+A:AB$ Effect | PrecostDesc$ :Gift of Chaos — | Cost$ 3 T | StaticAbilities$ GrantCascade | Triggers$ ExileEffect | SpellDescription$ The next noncreature spell you cast this turn has cascade. (When you cast that spell, exile cards from the top of your library until you exile a nonland card that costs less. You may cast it without paying its mana cost. Put the exiled cards on the bottom of your library in a random order.)
+SVar:GrantCascade:Mode$ Continuous | EffectZone$ Command | Affected$ Card.nonCreature+YouOwn | AffectedZone$ Stack | Execute$ ExileEff | AddKeyword$ Cascade | Description$ The next noncreature spell you cast this turn has cascade.
+SVar:ExileEffect:Mode$ SpellCast | EffectZone$ Command | Valid$ Card.nonCreature+YouOwn | AffectedZone$ Stack | Execute$ RemoveEffect | Description$ The next noncreature spell you cast this turn has cascade.
```

Comment:
Since this is just a helper you'll want this trigger to be `Static$ True` which will also make the Description$ pointless